_haikus/unimaginative_cloud.md

```diff
@@ -0,0 +1,8 @@
+--- layout: haiku
+title: Unimaginative Cloud
+author: pookzzz
+---
+
+Clouds float pass<br>
```

Comment:
This line is a little short at 3 syllables, add 2 more syllables here for it to be consistent with haiku format.